From: Mahalakshmi Kailasam <mahalakshmik@ami.com>
Date: Wed, 9 Sep 2026 00:00:00 +0000
Subject: [PATCH] Remove IdButton and NMIOut for TiogaPass

ID_BUTTON and NMI_OUT are not defined in the TiogaPass device tree's
gpio-line-names (arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-tiogapass.dts),
so power-control's find_line() for these fails and the service exits
with status 255, taking down State.Host0/Chassis0 with it.

Signed-off-by: Mahalakshmi Kailasam <mahalakshmik@ami.com>
---
 config/power-config-host0.json | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/config/power-config-host0.json b/config/power-config-host0.json
index e909e56..d238fbc 100644
--- a/config/power-config-host0.json
+++ b/config/power-config-host0.json
@@ -1,11 +1,5 @@
 {
     "gpio_configs": [
-        {
-            "Name": "IdButton",
-            "LineName": "ID_BUTTON",
-            "Type": "GPIO",
-            "Polarity": "ActiveLow"
-        },
         {
             "Name": "NMIButton",
             "LineName": "NMI_BUTTON",
@@ -13,12 +7,6 @@
             "Polarity": "ActiveLow"
         },
         {
-            "Name": "NMIOut",
-            "LineName": "NMI_OUT",
-            "Type": "GPIO",
-            "Polarity": "ActiveHigh"
-        },
-        {
             "Name": "PostComplete",
             "LineName": "POST_COMPLETE",
             "Type": "GPIO",
-- 
2.34.1
